snap/snapcraft.yaml

```diff
@@ -80,22 +154,40 @@ apps:
   curl:
     command: usr/bin/curl
     plugs: [network]
+  kong:
+    command: bin/kong-wrapper.sh
```

Comment:
Is there a reason kong is listed under apps when it's part of security-services?

Reply:
This is so folks could run kong commands manually, i.e. to get status checks of the gateway. It's only for debugging purposes and the wrapper script is somewhat complicated so providing an app here makes it easier to use.